src/App.tsx

```diff
@@ -1,24 +1,27 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+import "./App.css";
+import { TodoList } from "./interfaces/todo";
+import InputForm from "./components/InputForm";
 
 function App() {
+  const [todoList, setTodoList] = useState<TodoList>([]);
+
+  /*  function setItwm(item) {
```

Comment:
Now that we have a function example in InpuForm component, could you please remove this commented out code?